### ISSUE TITLE
Ubuntu / LibreOffice 7.2.5.2 / Error registering extension "Python cannot be instantiated"

### ISSUE THREAD

**Issue by johnjohndoe** [vmiklos/ged2dot#200]
**Environment**
- Ubuntu 20.04.3
- LibreOffice 7.2.5.2

**Reproducer steps**
1. Download `gedcom-7.3.oxt`
2. Install the extension in LibreOffice

**Actual result**
- The following error message pops up:

> ImplementationRegistration::registerImplementation() - The service com.sun.star.loader.Python cannot be instantiated /build/libreoffice-wlT1B7/libreoffice-7.2.5~rc2/stoc/source/implementationregistration/-implreg.cxx:1275

![libreoffice-error](https://user-images.githubusercontent.com/144518/153843395-0d722717-ec64-430e-b17e-ad0047c5fb2a.png)

**Expected behavior**
- The extension is installed and enabled.

**Comment by vmiklos** [comment]
I'll try to check with that exact version, but in the meantime, can you try if any other python-based LO extension work for you? Quite possibly the above error hints that some Python/LO packages are missing on your end, so I'm not sure this will be a ged2dot error.

**Comment by johnjohndoe** [comment]
You are right. I tried another plugin and it fails with the same error.

I figured out that `python-uno` is needed. I managed to install it via `sudo apt install libreoffice-script-provider-python` as mentioned [here](https://stackoverflow.com/a/69611420/356895).

You might want to mention this in the README.md. Other than that I think we can close this issue.

---

_... and I will try to figure out how to use the extension resp. install missing dependencies: only plain text is loaded by now_
